Deceiver.ts

```diff
@@ -22,27 +21,29 @@ export class DeceiverMirror<T, K extends keyof T> {
     }
 
     public getMethodNames (): K[] {
-        return _(this.getAllPrototypes())
+        return _.chain(this.getAllPrototypes())
             .map(prototype => ({prototype, names: Object.getOwnPropertyNames(prototype)}))
             .map(({prototype, names}) => ({
                 prototype,
                 names: names.filter(name => typeof Object.getOwnPropertyDescriptor(prototype, name).value == 'function'),
             }))
             .flatMap(({names}) => names)
             .filter(name => name != 'constructor')
```

Comment:
maybe you can use `!==` instead.

Reply:
It's already changed in PR with prettier to have `!=` in both `getMethodNames` and `getPropertyNames`